assemblies/installing-odh-v2.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv2
+
+[id="installing-odh-v2_{context}"]
+= Installing Open Data Hub (v2)
+
+[role='_abstract']
+This section describes how to install the Open Data Hub Operator v2.X on your OpenShift Container Platform from the OpenShift web console. 
+//For information about upgrading the Open Hub Operator v1.X to v2.X, see
+
+Installing Open Data Hub involves the following tasks:
+
+. Install the Open Data Hub Operator.
+. Install Open Data Hub components.
+. Access the Open Data Hub dashboard.
+
+[NOTE]
+--
+Open Data Hub Operator version 2.X represents an alpha release, only accessible on the *fast* channel. Later releases will change to the *rolling* channel when the operator attains greater stability.
```

Comment:
The original disclaimer on https://opendatahub.io/docs/quick-installation-new-operator says "later releases will transition to the **fast** channel" but I think this is supposed to be **rolling**?